packaging/leapp-supplements.spec

```
@@ -0,0 +1,111 @@
+# Define Globals
+%global leapp_datadir %{_datadir}/leapp-repository
+%global repositorydir %{leapp_datadir}/repositories
+%global custom_repositorydir %{leapp_datadir}/custom-repositories
+%global supplementsdir %{repositorydir}/system_upgrade_supplements
+
+%if 0%{?rhel} == 7
+    %define rpm_name leapp-supplements-el7toel8
+%endif
+%if 0%{?rhel} == 8
+    %define rpm_name leapp-supplements-el8toel9
+%endif
+
+# Define RPM Preamble
+Name:           %{rpm_name}
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        Custom actors for the Leapp project
+
+License:        MIT
+URL:            https://github.com/oamg/%{name}
```

I expect you want to have here
```suggestion
URL:            https://github.com/oamg/leapp-supplements
```